README.md

```diff
@@ -45,6 +45,15 @@ roles:
     become: true
 ```
 
+#### purge local modifications using appropriate variable
+
+```yaml
+selinux_all_purge: 1
+selinux_booleans_purge: 1
+selinux_fcontexts_purge: 1
+selinux_ports_purge: 1
+selinux_logins_purge: 1
+```
```

Comment:
There were some comments about using boolean True/False instead of numbers. Should this be the case?

Reply:
Fixed.

Reply:
having `selinux_all_purge: true` with the other `*purge` variables in the same yaml example is IMO confusing. Can you split it into two example files, one with `selinux_all_purge`, the other with the others?